channels/urbdrc/client/libusb/libusb_udevice.c

```diff
@@ -1482,8 +1482,8 @@ static void udev_free(IUDEVICE* idev)
 		log_libusb_result(urbdrc->log, WLOG_ERROR, "libusb_reset_device", rc);
 	}
 
-	/* release all interface and  attach kernel driver */
 #ifndef _WIN32
+	/* release all interface and  attach kernel driver */
```

Comment:
Wrong place, I´d move this define to `libusb_udev_attach_kernel_driver` and keep platform defines contained there.